### PR TITLE
Check for useless inherited timing points

### PR DESCRIPTION
**Pull request by cl8n**

### CODE REVIEW

File: checks/all modes/timing/CheckUnusedLines.cs
Comment (Naxesss) [inline checks/all modes/timing/CheckUnusedLines.cs]: Should be a separate paragraph since it's no longer about uninherited lines.

Would rather use something like this instead:
"Unused inherited lines don't cause any issues and are basically equivalent to bookmarks. Unless the mapper intended to do something with them (e.g. silencing slider ends/ticks), they can be safely removed, but removal is not necessary."

With the way it's written currently, the paragraph implies they should be changed because "there's no reason to keep them around", which is odd considering there's usually no reason to remove them either. It does improve file size by a few bytes, but I don't think it's a good idea to encourage micro-optimization.

File: checks/all modes/timing/CheckUnusedLines.cs
Comment (Naxesss) [inline checks/all modes/timing/CheckUnusedLines.cs]: Problems are reserved for issues that almost certinaly must be fixed (e.g. unrankables and game-breaking glitches). I'd at most make this a minor issue, anything else would be too intrusive for what it's worth.

File: checks/all modes/timing/CheckUnusedLines.cs
Comment (Naxesss) [inline checks/all modes/timing/CheckUnusedLines.cs]: Put continue on the next line for consistency, see [line 111](https://github.com/Naxesss/MapsetChecks/blob/master/checks/all%20modes/timing/CheckUnusedLines.cs#L111).

File: checks/all modes/timing/CheckUnusedLines.cs
Comment (Naxesss) [inline checks/all modes/timing/CheckUnusedLines.cs]: Remove the new line for consistency, see [line 81](https://github.com/Naxesss/MapsetChecks/blob/master/checks/all%20modes/timing/CheckUnusedLines.cs#L81).

File: checks/all modes/timing/CheckUnusedLines.cs
Comment (Naxesss) [inline checks/all modes/timing/CheckUnusedLines.cs]:
```suggestion
                        "{0} Inherited line changes nothing.",
```
Otherwise ambiguous for timestamps with both uninherited and inherited lines present.

Should also prepend "Uninherited line" for the other templates now that the issue message is simply "timing line" and not "uninherited line".

File: checks/all modes/timing/CheckUnusedLines.cs
Comment (Naxesss) [inline checks/all modes/timing/CheckUnusedLines.cs]: Was about to say `omitsBarLine` works in a different way to other properties and isn't properly handled here, but apparently it's unused for inherited lines (just tested), so you can just remove this comparison.

**Review by Naxesss — CHANGES_REQUESTED** [review]
Should be good to go after this. Have confirmed it works. It still considers inherited lines with no objects after them used (similarly only modifying SV but not having sliders), but I can add that part after merging.

File: checks/all modes/timing/CheckUnusedLines.cs
Comment (cl8n) [reply checks/all modes/timing/CheckUnusedLines.cs]: dont rly agree with the precedent but thisis ur program so Ok

**Pull request merged**
